### ISSUE TITLE
AWS Lambda

### ISSUE THREAD

**Issue by chadbrewbaker** [capitalist/cortera#1]
I have a need for a Cortera aware AWS Lambda function. Want me to make a contrib directory?

TODO List:
Ruby server side validations for all Cortera fields.
Elm/Javascript client side validations for all Cortera fields.
Cortera AWS Lambda zipfile generation script.
Lambda benchmarking. Cure any performance problems to lower the server bill.

**Comment by capitalist** [comment]
@chadbrewbaker That sounds like it'd be best for either a distinct client-side project, or a fork.

**Comment by chadbrewbaker** [comment]
Elm validations perhaps, but server side would make sense in gem.

**Comment by capitalist** [comment]
True, true. Well I'm happy to accept any PR's. We use this library for just the part of it that exists now, which is admittedly very lean. 

At the time I wrote it, I just needed proper exceptions for certain events so I could handle them.
